### DISCUSSION TITLE
Welcome to jarvis Discussions!

### DISCUSSION THREAD

**Discussion by kruthi-sb** [kruthi-sb/jarvis#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

 

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by ghost** [comment]
ok i seeing the changes now...
